change/@itwin-property-grid-react-f6c86575-936f-497c-a1f3-12d9f0d0f4b2.json

```
@@ -0,0 +1,7 @@
+{
+  "type": "minor",
+  "comment": "Visibility of PropertyGridWidget can now be controlled using a custom predicate.",
```

Let's be more explicit about what exactly was added.

```suggestion
  "comment": "Visibility of the widget, when creating it using `PropertyGridUiItemsProvider`, can now be controlled using a `propertyGridProps.shouldShow` predicate, which is invoked on every unified selection change. The default behavior stays the same - show the widget when a node or a non-transient element is selected.",
```